modules/custom/d_p_side_by_side/templates/paragraph--d-p-side-by-side.html.twig

```diff
@@ -50,7 +50,7 @@
     <div{{ attributes.addClass(classes) }}>
       {% block tiles_content %}
         <div class="d-p-tiles-wrapper expandable-content">
-          <div class="container items">
+          <div class="container-fluid items">
```

Comment:
Wstrzymalbym sie na chwile ze zmergowaniem bo mam wrażenie, że nie powinniśmy zmieniać tu ustawień containera - bo to zmieni zupełnie działanie paragrafu. Zatrzymajmy to na 2.01